mm2src/coins/nft/nft_structs.rs

```diff
@@ -749,16 +758,19 @@ impl NftCtx {
     #[cfg(not(target_arch = "wasm32"))]
     pub(crate) async fn lock_db(
         &self,
+        db_id: Option<&str>,
```

Comment:
Suggestion: use a new type DbId instead of str

Reply:
hmmm, are you concern because of `shared_db_id` ?

Reply:
I mean to use a type like `struct DbId(String)` to ensure the correct type everywhere in code
as in the 'New Type' idiom

Reply:
Is this really needed ? I think the method name is already very expressive imo. 

`lock_db` with given `db_id`